insights/parsers/dmidecode.py

```diff
@@ -110,7 +110,8 @@ class DMIDecode(Parser, LegacyItemAccess):
     """
     Class for DMI information.
     """
-
+    # TODO:
+    # Remove below 4 dicts
```

Comment:
@bfahr - Thanks a lot. I updated the code according to your suggestions, please check it again.